docker/docker-compose.host.yml

```
@@ -17,6 +17,7 @@ services:
     environment:
       EDITOAST_PORT: 8090
       OSRD_BACKEND_URL: "http://localhost:8080"
+      OSRDYNE_API_URL: "http://127.0.0.1:4242"
```

nitpick: just for consistency with above and below?

```suggestion
      OSRDYNE_API_URL: "http://localhost:4242"
```